Algea ground intake PR #2

AndreTommySalehi opened the pull request.

M-i-c-h-a-e-l-599 committed:
All Files in folder created

All files created

M-i-c-h-a-e-l-599 committed:
Debug

Did more Debugging and almost done with the Rollers

M-i-c-h-a-e-l-599 committed:
Rollers and MotorLog

Rollers is basically completed with one more error to sort out. I have created a MotorLog for DogLog and will start on the wrist after the mistake on Rollers is corrected.

M-i-c-h-a-e-l-599 committed:
Rollers is done

debugging is done for rollers. Now time for Wrist 💯

M-i-c-h-a-e-l-599 committed:
Wrist update

Wrist is basically done. Just got to correct some stuff.

M-i-c-h-a-e-l-599 committed:
Complete

All I need to do now is to test it on the board and then make the corrections. 😧

M-i-c-h-a-e-l-599 committed:
Complete again

Added things

M-i-c-h-a-e-l-599 committed:
Code Code Code work :)

Code worked

M-i-c-h-a-e-l-599 committed:
added some stuff

AndreTommySalehi commented on 2025_NoName/src/main/java/frc/robot/Subsystems/AlgaeGroundIntake/Wrist/WristIO.java:
```diff
@@ -0,0 +1,89 @@
+
+public interface WristIO {
+    public abstract class WristIOInputs {
+        protected double angularRads = 0.0;
+        protected double getAngularVelocityRadPerSec = 0.0;
+        protected double appliedVoltage = 0.0;
+        protected double setpointAngleRads = 0.0;
+        protected double currentAmps = 0.0;
+        protected double tempCelcius = 0.0;
```
get rid of ur supers cus you already log everything in MotorLog and SimLog without it. just comment it out

Meerlikeskeer replied:
replace them with the MotorLog.log("Wrist", motorName); in real layer and SimLog.log("Wrist", motorName); in the sim layer.

AndreTommySalehi commented on 2025_NoName/src/main/java/frc/robot/Subsystems/AlgaeGroundIntake/Rollers/RollersIO.java:
```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.algaegroundintake.rollers;
+
+public abstract class RollersIO {
+    protected double velocityRadsPerSec = 0.0;
+    protected double appliedVoltage = 0.0;
+    protected double velocitySetpoint = 0.0;
+    protected double currentAmps = 0.0;
+    protected double tempCelcius = 0.0;
```
Comment out cus you already log all of ur inputs in MotorLog and SimLog without it.

Meerlikeskeer replied:
replace them with the MotorLog.log("Wrist", motorName); in real layer and SimLog.log("Wrist", motorName); in the sim layer.

M-i-c-h-a-e-l-599 committed:
Fixed Problems

Fixed the comments that Andre told me to change.

M-i-c-h-a-e-l-599 committed:
Updated the PR suggestions

Replaced the updateinputs with a Simlog for both roller and wrist.

M-i-c-h-a-e-l-599 committed:
Mechanism2d Update

Moved my Mechanism 2d out of the IO Sim layer. Made an intake commands folder, not done with it yet.
